keyboards/irene/keymaps/default/config.h

```diff
@@ -0,0 +1,10 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
+
+#define USE_I2C
+#define SSD1306OLED
+#define OLED_ROTATE180
+
+#endif
```

Comment:
Is there a specific reason this file exists instead of including these macros in the `config.h` two directories up?